projects/configs/despiad.yaml

```diff
@@ -0,0 +1,32 @@
+#  Copyright (c) 2024 University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+project:
+    name: "despiad"
+    modalities:
+      - "CT"
```

Comment:
with CT AFAIK there's some extra fun parts which have burned in data. I think to avoid this we may want to have:


- Minimum instances in a series. At the project level can configure minimum instances in series so that we can filter out a single screenshot that may well still have the `CT` modality
- Series filter at a manufacturer level in the project conifg. This is because certain series numbers can always have burned in data - Worth asking John Watts about an example of this
- Related, but to have an allowed manufacturer regex list in the config. So that you filter out manufacturers which you don't know you can handle - in case there's more burned in data

Reply:
oh fun! Yeah they sound like good ideas. Have you ever used something like [deid](https://github.com/pydicom/deid) or [presidio](https://github.com/microsoft/presidio) to remove burned in data?